### ISSUE TITLE
Canadian archeological radiocarbon database

### ISSUE THREAD

**Issue by sckott** [ropensci/archeology#4]
http://www.canadianarchaeology.ca/localc14/c14results.php

Seems no way to bulk download/ftp, or API

**Comment by JeffMoy** [comment]
Hi Sckott!
Did you find a way to get access to the database?
Thanks
Jeff

**Comment by sckott** [comment]
Thanks for the ping. I don't remember now, been too long. @benmarwick do you know if there's a way to access it?

**Comment by benmarwick** [comment]
I don't know of a way using R. I see on the website it says "Accessing CARD's full capabilities requires a security-account available only to researchers at accredited institutions." So credentials are required, it seems.

**Comment by sckott** [comment]
Yeah, doesn't sound promising

**Comment by JeffMoy** [comment]
Yeah that's what I feared. I though may be you knew someone having credential or the database on his repository.
Thank you guys
